frontend/src/Home.test.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { render, unmountComponentAtNode } from "react-dom";
+import { act } from "react-dom/test-utils";
+import Home from "./Home";
+
+let container = null;
+beforeEach(() => {
+  container = document.createElement("div");
+  document.body.appendChild(container);
+});
+
+afterEach(() => {
+  unmountComponentAtNode(container);
+  container.remove();
+  container = null;
+});
+
+it("renders correct welcome message", () => {
```

Comment:
do we want to split this snapshot test into two separate "tests"

see https://docs.google.com/document/d/1WqjdTF5ifkBx-9Ls3T1V4reAdnh0DRe_WZcvYlmnBlk/edit#heading=h.544ix87lupkx.

Reply:
done